apps/web/app/api/webhooks/retell-ai/route.ts

```diff
@@ -172,9 +172,7 @@ async function handleCallAnalyzed(callData: RetellCallData) {
     log.info(`Processing web call ${call_id} for agent ${agent_id}, user ${userId}, team ${teamId}`);
   } else {
     const phoneNumberRepo = new PrismaPhoneNumberRepository(prisma);
-    const phoneNumber = await phoneNumberRepo.findByPhoneNumber({
-      phoneNumber: from_number,
-    });
+    const phoneNumber = await phoneNumberRepo.findByPhoneNumberForCallAnalyze(from_number);
```

Comment:
We shouldn't add business logic in method's name. More info:- https://calendso.slack.com/archives/C08B8SYUY02/p1760351227076169

Reply:
Fair, will address.